kubernetes/samples/scripts/terraform/terraformrc

```diff
@@ -0,0 +1,4 @@
+# Example of terraform plugin file
+providers {
+ oci = "${HOME}/.terraform.d/plugins/terraform-provider-oci"
```

Comment:
actually, this may no longer be required.  The OCI plugin can now be automatically downloaded when you do "terraform init" as long as the scripts contain a "provider "oci"" reference.